src/AdminConsole/Program.cs

```diff
@@ -189,5 +194,6 @@ void RunTheApp()
     app.MapPasswordless()
         .LoginRoute?.AddEndpointFilter<LoginEndpointFilter>();
     app.MapRazorPages();
+    app.MapRazorComponents<App>();
```

Comment:
I wonder how Blazor/RazorComponents will be affected by current stack, e.g. Middlewares and Auth. Most things I expect to "just work", but we need to tweak auth-policies to cover components like we've done with Pages.